attributes/default.rb

```diff
@@ -181,6 +181,10 @@
 default['datadog']['histogram_aggregates'] = 'max, median, avg, count'
 default['datadog']['histogram_percentiles'] = '0.95'
 
+# Multiple endpoints/api_keys settings
```

Comment:
What do you think about specifying in the comment which version of the Agent these parameters will start working with? (we never do it in this file but it would be helpful IMO)